Vagrantfile

```diff
@@ -147,6 +152,14 @@ Vagrant.configure(2) do |config|
   end
 
   config.vm.provider "parallels" do |prl, override|
+    # Default box for parallels on arm64
+    unless ENV.has_key?('VAGRANT_BOX')
+      arch = `uname -m`.chomp
+      if (arch == 'arm64' || arch == 'aarch64')
+        override.vm.box = "bento/ubuntu-20.04-arm64"
```

Comment:
From the other comment, with this PR, we would define separate boxes in 3 different places without any close proximity to each other, which would certainly lead to a potential problem someone updates only the base image to use ubuntu-22.04, but leaves parallels and libvrt alone, and then something accidentally breaks.

I would much prefer to see that we define at the top of this file something like:

```
# All images should point to the same version of ubuntu
base_image = "bento/ubuntu-20.04"
base_parallels_image = if (arch == 'arm64' || arch == 'aarch64')
  "#{base_image}-arm64"
else
  base_image
end
base_libvrt_image = "generic/ubuntu2004"
```

This creates a much stronger spatial link between these box names, and should make it harder to have them differ in ubuntu verison.

Within the code, we'd use `config.vm.box = base_image`, `override.vm.box = base_parallels_image`, etc.